### PR TITLE
Implement FAQ sheets fetcher

### LINKED ISSUE #65: Implement FAQ Sheets fetcher

### PR DESCRIPTION
**Pull request by andriawan**
closes #65

Implement sheets fetcher to fetch FAQ database from https://kcov.id/wbw-faq

The fetched content will be stored in data/wbw-faq-sheets.json with the following structure:

```
[
   // based on column in the spreadsheet
  {
    "kategori_pertanyaan": "Tatalaksana Isoman",
    "pertanyaan": "Berapa lama isolasi mandiri?",
    "jawaban": "TANPA GEJALA: 10 hari pasca hasil tes nya keluar.DENGAN GEJALA: 10 hari dari hari pertama muncul gejala, dan 3 hari terakhir harus bebas gejala. Jika gejala lebih dari 10 hari, isolasi dilanjutkan hingga gejala hilang ditambah dengan 3 hari bebas gejala. ",
    "created_date": "10 Jul 2021",
    "sumber": "Protokol Tata Laksana COVID-19, Buku Saku Ed. 2",
    "link": "https://covid19.go.id/p/protokol/protokol-tatalaksana-covid-19-di-indonesia",
    "published_date": "08 Feb 2021"
  },
]
```

### CODE REVIEW

**Comment by andriawan** [comment]
please just review this PR mas @zainfathoni

File: etc/fetchers/fetch-faq-sheet.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const cheerio = require("cheerio");
+const fs = require("fs");
+const path = require("path");
+const fetch = require("cross-fetch");
+const FAQ_LINK = "https://kcov.id/wbw-faq";
+
+module.exports.fetchFaqSheet = async function fetchFaqSheet() {
```
Comment (zainfathoni) [inline etc/fetchers/fetch-faq-sheet.js:7]:
```suggestion
module.exports.fetchFaqSheets = async function fetchFaqSheets() {
```

For consistency, please always use the plural form of `Sheets` everywhere, including the name of this file.
Please rename the file into `fetch-faq-sheets.js`.

File: etc/fetchers/fetch-faq-sheet.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        created_date: $(row.get(3)).text(),
+        sumber: $(row.get(4)).text(),
+        link: $(row.get(5)).text(),
+        published_date: $(row.get(6)).text(),
+      };
+    })
+    .toArray();
+
+  fs.writeFileSync(
+    path.resolve(__dirname, "../../data/wbw-faq-sheet.json"),
```
Comment (zainfathoni) [inline etc/fetchers/fetch-faq-sheet.js:27]:
```suggestion
    path.resolve(__dirname, "../../data/wbw-faq-sheets.json"),
```

File: etc/fetchers/fetch-wbw.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const { fetchDocs } = require("./fetch-docs");
 const { fetchSheets } = require("./fetch-sheets");
+const { fetchFaqSheet } = require("./fetch-faq-sheet");
```
Comment (zainfathoni) [inline etc/fetchers/fetch-wbw.js:3]:
```suggestion
const { fetchFaqSheet } = require("./fetch-faq-sheets");
```

File: etc/fetchers/fetch-wbw.js [hunk 4]
```diff
@@ -8,6 +9,9 @@ function fetchWbw() {
 
   console.log("Fetching sheets");
   fetchSheets().then(() => console.log("DONE fetching sheets"));
+
+  console.log("Fetching Faq sheets");
+  fetchFaqSheet().then(() => console.log("DONE fetching faq sheets"));
```
Comment (zainfathoni) [inline etc/fetchers/fetch-wbw.js:14]:
```suggestion
  console.log("Fetching FAQ sheets");
  fetchFaqSheet().then(() => console.log("DONE fetching FAQ sheets"));
```

**Review by zainfathoni — CHANGES_REQUESTED** [review]
Thanks for contributing, @andriawan!
Overall, it looks good to me.

I'm just nitpicking minor spelling inconsistencies. 😅 🙏

File: etc/fetchers/fetch-faq-sheet.js (hunk 1 above)
Comment (andriawan) [reply etc/fetchers/fetch-faq-sheet.js:7]: oke mas @zainfathoni. will rename it soon

**Comment by andriawan** [comment]
ready to re-check mas @zainfathoni

**Review by zainfathoni — APPROVED** [review]
LGTM 💯

**Pull request merged**